<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSetting</class>
 <widget class="QDialog" name="DialogSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>931</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="GB_SETTING">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>971</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>함초롬돋움</family>
     <pointsize>20</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox  {
    margin-top: 10px;
	font: 20pt &quot;함초롬돋움&quot;;
}

QGroupBox::title  {
    subcontrol-origin: margin;
    subcontrol-position: top center;
    padding: 10px 8000px 10px 8000px;
    background-color: #595959;
    color: rgb(255, 255, 255);
}
QLabel {
	font: 10pt &quot;함초롬돋움&quot;;
}
QComboBox{
	font: 10pt &quot;함초롬돋움&quot;;
	background-color: #ffffff
}
QPushButton{
	font: 10pt &quot;함초롬돋움&quot;;
	background-color: #ffffff
}
QLineEdit{
	font: 10pt &quot;함초롬돋움&quot;;
	background-color: #ffffff
}



</string>
   </property>
   <property name="title">
    <string>설정</string>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>381</width>
      <height>821</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>에러</string>
     </attribute>
     <widget class="QLabel" name="label_148">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>함초롬돋움</family>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <italic>false</italic>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>장비 에러</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>251</width>
        <height>306</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="4" column="0">
        <widget class="QLabel" name="label_152">
         <property name="font">
          <font>
           <family>함초롬돋움</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>종이컵2</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_158">
         <property name="font">
          <font>
           <family>함초롬돋움</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>아이스크림</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="BTN_ERROR_CLEAR_PAPER_CUP_1">
         <property name="text">
          <string>에러 클리어</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QLineEdit" name="LE_DEVICE_ERROR_ICECREAM">
         <property name="font">
          <font>
           <family>함초롬돋움</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label_150">
         <property name="font">
          <font>
           <family>함초롬돋움</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>제빙기1</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="9" column="2">
        <widget class="QPushButton" name="BTN_ERROR_CLEAR_ICECREAM">
         <property name="text">
          <string>에러 클리어</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QPushButton" name="BTN_ERROR_CLEAR_PET_CUP_1">
         <property name="text">
          <string>에러 클리어</string>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_156">
         <property name="font">
          <font>
           <family>함초롬돋움</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>온수기</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="label_155">
         <property name="font">
          <font>
           <family>함초롬돋움</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>PET컵2</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="LE_DEVICE_ERROR_HOT_WATER">
         <property name="font">
          <font>
           <family>함초롬돋움</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="LE_DEVICE_ERROR_ICE_1">
         <property name="font">
          <font>
           <family>함초롬돋움</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_154">
         <property name="font">
          <font>
           <family>함초롬돋움</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>PET컵1</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="LE_DEVICE_ERROR_PAPER_CUP_2">
         <property name="font">
          <font>
           <family>함초롬돋움</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="LE_DEVICE_ERROR_PET_CUP_2">
         <property name="font">
          <font>
           <family>함초롬돋움</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="BTN_ERROR_CLEAR_ICE_1">
         <property name="text">
          <string>에러 클리어</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="LE_DEVICE_ERROR_PET_CUP_1">
         <property name="font">
          <font>
           <family>함초롬돋움</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLineEdit" name="LE_DEVICE_ERROR_SODA">
         <property name="font">
          <font>
           <family>함초롬돋움</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_149">
         <property name="font">
          <font>
           <family>함초롬돋움</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>커피머신</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_153">
         <property name="font">
          <font>
           <family>함초롬돋움</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>종이컵1</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QPushButton" name="BTN_ERROR_CLEAR_HOT_WATER">
         <property name="text">
          <string>에러 클리어</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QPushButton" name="BTN_ERROR_CLEAR_PAPER_CUP_2">
         <property name="text">
          <string>에러 클리어</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QPushButton" name="BTN_ERROR_CLEAR_PET_CUP_2">
         <property name="text">
          <string>에러 클리어</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="LE_DEVICE_ERROR_COFFEE">
         <property name="font">
          <font>
           <family>함초롬돋움</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="QPushButton" name="BTN_ERROR_CLEAR_SODA">
         <property name="text">
          <string>에러 클리어</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="LE_DEVICE_ERROR_PAPER_CUP_1">
         <property name="font">
          <font>
           <family>함초롬돋움</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QPushButton" name="BTN_ERROR_CLEAR_COFFEE">
         <property name="text">
          <string>에러 클리어</string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="label_157">
         <property name="font">
          <font>
           <family>함초롬돋움</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>탄산수</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_151">
         <property name="font">
          <font>
           <family>함초롬돋움</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>제빙기2</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="LE_DEVICE_ERROR_ICE_2">
         <property name="font">
          <font>
           <family>함초롬돋움</family>
           <pointsize>10</pointsize>
           <weight>50</weight>
           <italic>false</italic>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="BTN_ERROR_CLEAR_ICE_2">
         <property name="text">
          <string>에러 클리어</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="GB_SUPER_STOP">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>440</y>
        <width>361</width>
        <height>241</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox  {
    border: 1px solid gray;
    border-color: #FF5D1717;
    margin-top: 10px;
    font-size: 12px;
    border-radius: 1px;
}

QGroupBox::title  {
    subcontrol-origin: margin;
    subcontrol-position: top center;
    padding: 5px 8000px 5px 8000px;
    background-color: #FF5D1717;
    color: rgb(255, 255, 255);
}</string>
      </property>
      <property name="title">
       <string>SUPER STOP</string>
      </property>
      <widget class="QLabel" name="label_203">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>60</y>
         <width>251</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>함초롬돋움</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Board Connection Error (LAN2CAN)</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="LE_SUPER_STOP_BOARD_CONNECTION">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>함초롬돋움</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_209">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>90</y>
         <width>251</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>함초롬돋움</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Robot Data Error</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="LE_SUPER_STOP_ROBOT_DATA">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>함초롬돋움</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_210">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>120</y>
         <width>251</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>함초롬돋움</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Robot Miss Command Check Error</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="LE_SUPER_STOP_ROBOT_MISS_COMMAND_CHECK">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>함초롬돋움</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="LE_SUPER_STOP_OUTLET_CONNECTION">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>함초롬돋움</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_211">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>150</y>
         <width>251</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>함초롬돋움</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Outlet Connection Error</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_212">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>180</y>
         <width>251</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>함초롬돋움</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Outlet Control Error</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="LE_SUPER_STOP_OUTLET_CONTROL">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>180</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>함초롬돋움</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_213">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>210</y>
         <width>251</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>함초롬돋움</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Outlet Sensor Error</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="LE_SUPER_STOP_OUTLET_SENSOR">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>210</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>함초롬돋움</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLabel" name="label_214">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>30</y>
         <width>251</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>함초롬돋움</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Platform Open Error</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="LE_SUPER_STOP_PLATFORM_OPEN">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>함초롬돋움</family>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>테스트</string>
     </attribute>
     <widget class="QTableWidget" name="TW_TEST_MENU">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>331</width>
        <height>371</height>
       </rect>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>140</number>
      </attribute>
     </widget>
     <widget class="QPushButton" name="BTN_TEST_ORDER">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>410</y>
        <width>89</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>테스트 주문</string>
      </property>
     </widget>
     <widget class="QPushButton" name="BTN_DB_RELOAD">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>680</y>
        <width>89</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>DB reload</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="CB_ADD_SHOT">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>420</y>
        <width>92</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>add Shot</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
